src/tensor.cpp

```diff
@@ -458,6 +458,22 @@ std::size_t TensorDescriptor::GetNumBytes() const
 
 bool TensorDescriptor::IsPacked() const { return this->packed; }
 
+bool TensorDescriptor::IsContiguous() const
+{
+    size_t plane_size    = 1;
```

Comment:
use int64_t for plane_size

Reply:
This code is copied from new commit in upstream repo. I also see some other thing that need to be fixed here (for example static_cast<int64_t> is not necessary in line 472). But let's wait for when we can update to latest commit in upstream repo then I will fix this function based on the function on upstream repo.